src/operator.go

```diff
@@ -33,6 +37,7 @@ type Querier[T any] interface {
 		ctx context.Context,
 		collection *Collection,
 		filter map[string]any,
+		sorting bson.D,
```

Comment:
Again might be dangerous to be so implementation specific in a generic component